solutions/go/02-cz2/diff/cmd/myshell/main.go.diff

```diff
@@ -0,0 +1,20 @@
+@@ -1,17 +1,18 @@
+ package main
+
+ import (
+ 	"bufio"
+ 	"fmt"
+ 	"os"
+ )
+
+ // Ensures gofmt doesn't remove the "fmt" import in stage 1 (feel free to remove this!)
+ var _ = fmt.Fprint
+
+ func main() {
+ 	fmt.Fprint(os.Stdout, "$ ")
+
+ 	// Wait for user input
+-	bufio.NewReader(os.Stdin).ReadString('\n')
++	command, _ := bufio.NewReader(os.Stdin).ReadString('\n')
```

Comment:
@andy1li not sure if we should be ignoring errors - for an official solution, users might expect us to be a bit more "production-ready".